src/Service/Modules/TariffModule.cs

```diff
@@ -38,8 +43,8 @@ private object GetTariffs()
         }
 
         private object GetTariff(int id)
```

Comment:
I think you'll need to add the privileges to the GetAll option too. That will allow the create button on the index page to be disabled where appropriate

Reply:
👍